curriculumcourse/pom.xml

```diff
@@ -36,26 +35,39 @@
   <packaging>kjar</packaging>
   <name>Course Scheduling Example</name>
   <description>University course scheduling using Planner. Assigns lectures to rooms.</description>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
   <dependencies>
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>kie-api</artifactId>
-      <scope>provided</scope>
+      <version>${version.org.kie}</version>
     </dependency>
     <dependency>
       <groupId>org.optaplanner</groupId>
       <artifactId>optaplanner-core</artifactId>
-      <scope>provided</scope>
+      <version>${version.org.optaplanner}</version>
```

Comment:
Where is there `${version.org.optaplanner}` coming from?
I don't see it in this pom, nor in the master pom.

- if this pom extends kie-wb-playground parent pom which extends kie-parent pom, then we don't need the <version> tag at all, because dependencyManagement is inherited
- if this pom does not indirectly extend kie-parent pom, we need a <properties> element that defines `${version.org.optaplanner}`. Thats a pita for Micheal Biarnes's release though, so I 'd much rather prefer the previous solution.